golang/cmd/dagent/.env.example

```diff
@@ -41,3 +40,4 @@ UPDATE_HOST_TIMEZONE=true
 UPDATE_METHOD=off
 UPDATE_POLL_INTERVAL=600s
 WEBHOOK_TOKEN=
+SECRET_PRIVATE_KEY_FILE=/path/to/secret.key
```

Comment:
What was the motivation to move the env end of the file?

Reply:
Because `SECRET_PRIVATE_KEY_FILE` is now only for `# DAgent specific options`.